src/main/java/com/dnd/runus/auth/oidc/provider/JwtParser.java

```diff
@@ -25,7 +25,7 @@ public static Map<String, String> parseHeaders(ObjectMapper objectMapper, String
         try {
             return objectMapper.readValue(Base64.getUrlDecoder().decode(header), new TypeReference<>() {});
         } catch (IOException e) {
-            throw new BusinessException(ErrorType.FAILED_PARSING, e.getMessage());
+            throw new BusinessException(ErrorType.INVALID_BASE64, e.getMessage());
```

Comment:
응답 에러 메세지에는 많은 정보가 포함되어 있으면 좋지 않다고 생각해요!
보안 상의 문제라든지(서버의 내부 구조, db 쿼리 등)
 그래서 상세 에러는 log로 남기고, 
클라이언트에 응답되는 메세지는 최소한의 정보만 리턴한는게 좋다고 생각하는데 이렇게 하면 어떠신가요?
```suggestion
            log.warn("{}; Unsupported token[{}];", e.getMessage(), token);
            throw new BusinessException(ErrorType.INVALID_BASE64, "Unsupported Id token: " + token);
```

Reply:
로깅을 하는 클래스가 많으면 같은 로그를 여러 번 적는 등 중복되는 경우가 생길 수 있을 것 같아요
그래서 로깅은 최대한 `ExceptionRestHandler`에서만 처리하는 게 좋을 것 같아요

그리고 message에 token을 적으면 민감한 정보를 노출하니 좋지 않을 것 같아요

대신 `ErrorType.INVALID_BASE64` 만 사용하던가 단순한 메세지만 남기면 좋을 것 같아요

Reply:
단순한 메세지도 같이 남기면 좋을 것 같아요!

Reply:
`e.getMessage()` 대신 `OIDC header를 디코딩하는데 실패했습니다.`라는 메세지로 수졍했어요